src/GHC/Util.hs

```diff
@@ -241,6 +247,26 @@ isCommentMultiline :: Located AnnotationComment -> Bool
 isCommentMultiline (L _ (AnnBlockComment _)) = True
 isCommentMultiline _ = False
 
+declName :: HsDecl GhcPs -> Maybe String
```

Comment:
RIP the comment? :(

Reply:
Sorry @googleson78 something wrong? What do you mean here?

Reply:
In #710 I added a bit of a comment to the function. It seems to be red in the diff for `Utils.hs` here? I'm guessing something while merging/rebasing?

Reply:
https://github.com/ndmitchell/hlint/pull/726/files#diff-fbbb414b66284e60d388bd81dc2b606cL277
This part here

Reply:
Oh! I see what you mean. I have no idea how that happened. Sorry, restored.

Reply:
Thanks! <3